Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -458,13 +462,9 @@ def prepare_enforcement_boundary_create_output(response: Dict, already_exists: b
     headers = list(hr_outputs.keys())
 
     if already_exists:
-        title = "Enforcement Boundary {} already exists:\n".format(response.get("href"))
-    else:
-        title = "Enforcement Boundary:\n"
+        demisto.info("Enforcement boundary {} already exists.".format(response.get("name")))
```

Comment:
There's no need to have this log line in the prepare HR functions. Remove the `already_exists` params from these functions and just log directly from the command function.

Reply:
Okay, I've added the log in the command function itself